src/QsCompiler/Compiler/CompilationLoader.cs

```diff
@@ -329,14 +329,15 @@ public CompilationLoader(SourceLoader loadSources, ReferenceLoader loadReference
             foreach (var diag in this.VerifiedCompilation?.Diagnostics() ?? Enumerable.Empty<Diagnostic>())
```

Comment:
nit:  I think the changes here are no longer needed, and I tend to try and limit the scope of defined things if possible.